### ISSUE TITLE
{Linked:Bug43225590; LinkedBug}[Voice access - Adaptive Cards]: Voice access user is not able to access the Options present under the combo box using Show Numbers commands.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7932]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:

OS Build: 25201.1000
Edge dev: Version 107.0.1379.1 (official build) dev (64-bit)

Application URL: [Schema Explorer | Adaptive Cards](https://adaptivecards.io/explorer/)

Accessibility Tool: Voice Access

Pre-requisite:
OS Settings->Accessibility->Voice Access (Turn ON)

Repro Steps:
Launch the above-mentioned URL on edge Dev with Valid credentials.
Command: "Voice access wake up"
Command: "Click Input.Choiceset".
Command: "Expand What color do you want" combo box.
Try to select the options from the combo box using VA commands.
Observe the issue.
Actual Result:
User is not able to select the options from the combo box using VA commands.

Expected Result:
User should be able to select the options present under the combo box using VA commands.

Note:
Issue is existing for all the combo boxes and options present under the adaptive cards.

User Experience:
Voice access users will get impact and will get disoriented if this behavior persists.

MAS Reference:
[No Disruption of Accessibility Features - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01)

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/104482612/192795678-141a705f-4214-45b5-8032-8c53f5a500ce.mp4

<img width="960" alt="Code snippet of Combo box options" src="https://user-images.githubusercontent.com/104482612/192795706-8203e660-13f8-4d32-a795-199e4be94abc.png">

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @Susmitha474!

I am closing this issue for the same reasons as https://github.com/microsoft/AdaptiveCards/issues/7505.

We are utilizing a basic HTML element here, and the same issue can be seen on the [Mozilla site](https://developer.mozilla.org/en-US/docs/web/html/element/select).

This bug should be filed with the Windows Voice Access team.

Let me know if you have any questions!

**Comment by KarunaNarang** [comment]
Tracking with https://microsoft.visualstudio.com/OS/_workitems/edit/43225590

**Comment by vagpt** [comment]
As per Anna comment adding A11yTTValidated tag.
